byon/files/seaclouds-catalog.bom

```diff
@@ -333,8 +329,6 @@ brooklyn.catalog:
             deployer.port: $brooklyn:component("deployer").attributeWhenReady("brooklynnode.webconsole.httpPort")
             sla.host: $brooklyn:component("sla-core").attributeWhenReady("host.address")
             sla.port: $brooklyn:component("sla-core").attributeWhenReady("http.port")
-            monitor.manager.host: $brooklyn:component("monitor-manager").attributeWhenReady("host.address")
-            monitor.manager.port: $brooklyn:component("monitor-manager").config("monitor.manager.port")
```

Comment:
I cannot evaluate the correctness of the .bom. 
Can you give some insight about why the monitor.manager is being removed from here?